datafusion/core/src/execution/session_state_defaults.rs

```diff
@@ -119,6 +120,11 @@ impl SessionStateDefaults {
         functions_window::all_default_window_functions()
     }
 
+    /// returns the list of default [`TableFunction`]s
+    pub fn default_table_functions() -> Vec<Arc<TableFunction>> {
+        functions_table::all_default_table_functions()
+    }
+
```

Comment:
Can we get the defaults from a singleton ? Like other default getters, with get_or_init or smth similar

Reply:
This is a good idea, updated